.github/workflows/update_cve_database.yml

```
@@ -0,0 +1,61 @@
+          pip install cve-bin-tool==3.4
+
+      - name: Update CVE database
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 40
+          max_attempts: 3
+          retry_wait_seconds: 300
+          command: |
+            source "${{ env.CVE_DIR }}/env/bin/activate"
+            python3 scripts/update_cve_db.py
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)